src/server.cpp

```diff
@@ -2260,6 +2248,10 @@ void Server::fadeSound(s32 handle, float step, float gain)
 		m_playing_sounds.erase(it);
 }
 
+// FIXME: Yet unused because removing sounds from objects that go out of range
+// (client side) would need to be played back again when coming into range again.
+// Currently, the client will initiate m_playing_sounds clean ups indirectly by
+// "Server::handleCommand_RemovedSounds".
```

Comment:
why keep dead code when we have it in the Git history?

Reply:
I thought it would be easier to rebase related PRs, and the code is presumably only dead for a short time. At the risk that a "short time" turns into years, I removed it.